src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/BeneficialOwnersStatementType.java

```diff
@@ -3,29 +3,36 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 
 public enum BeneficialOwnersStatementType {
+  ALL_IDENTIFIED_ALL_DETAILS("all_identified_all_details"),
+  SOME_IDENTIFIED_ALL_DETAILS("some_identified_all_details"),
+  NONE_IDENTIFIED("none_identified");
 
-    ALL_IDENTIFIED_ALL_DETAILS("all_identified_all_details"),
-    SOME_IDENTIFIED_ALL_DETAILS("some_identified_all_details"),
-    NONE_IDENTIFIED("none_identified");
+  private final String beneficialOwnersStatement;
 
-   private final String beneficialOwnersStatement;
+  BeneficialOwnersStatementType(String beneficialOwnersStatement) {
+    this.beneficialOwnersStatement = beneficialOwnersStatement;
+  }
 
-   BeneficialOwnersStatementType(String beneficialOwnersStatement) {
-        this.beneficialOwnersStatement = beneficialOwnersStatement;
+  /**
+   * Json creator allows non-matching enum values to be interpreted by the method which determines
+   * them as null
+   *
+   * @param beneficialOwnersStatement
+   * @return
+   */
+  @JsonCreator
+  public static BeneficialOwnersStatementType findByBeneficialOwnersStatementTypeString(
+      String beneficialOwnersStatement) {
+    for (BeneficialOwnersStatementType type : values()) {
```

Comment:
It's not a type - a statement

Reply:
Think I miss-understood what this was. Use: Enum.ValueOf

Reply:
This is not my code @AndreKramer7654 Andre . May be somebody had a different purpose of it

Reply:
Yeah, it just got reformated so marked in PR.